project.py

```diff
@@ -7,4 +7,11 @@
 current_time = datetime.datetime.now()
 
 # Printing value of now.
-print("Time now at greenwich meridian is:", current_time)
\ No newline at end of file
+print("Time now at greenwich meridian is:", current_time)
+
+#Printing out the distance between 19/07/23 and 01/01/24 
+from datetime import date
+f_date = date(2023, 7, 19)
+l_date = date(2024, 1, 1)
```

Comment:
Why did you choose the variable names f_date and l_date?


maybe we can give them names like `from_date` and `to_date`

Reply:
Will fix as part of following PR